coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -33,3 +38,83 @@ def __init__(self, documentation, language, docstyle,
 
     def __str__(self):
         return self.documentation
+
+    def parse(self):
+        """
+        Parses documentation independent of language and docstyle.
+
+        :return:
+            A list of the parsed documentation metadata.
+        :raises NotImplementedError:
+            When no parsing method is present for the given language and
+            docstyle.
+        """
+        if self.language == "python" and self.docstyle == "default":
+            return self._parse_documentation_with_symbols((":param ", ": "),
+                                                          ":return:")
```

Comment:
uh yes but I am not sure if I should do that, what if the user doesn't put a space after `:return:`
like:
```
:return:Blah
```

then the parsing would fail.
This will need more discussion.